### PR TITLE
Custom Apps/Games

### PR DESCRIPTION
**Pull request by rxfe-a**
This PR adds the ability to add custom games and custom apps

### CODE REVIEW

File: static/apps.html [hunk 1]
```diff
@@ -27,6 +27,8 @@
      </div></div></div></div>
   <!-- Nav Bar End-->
     <div class="input-container">
+      <button id="add" class="sidebarBTN">Add Custom Apps</button>
```
[the remark this answers is not in the document]
Comment (rxfe-a) [reply static/apps.html:30]: Like in the game-container?

File: static/assets/scripts/apps.js [hunk 2]
```diff
@@ -289,6 +289,60 @@ document.addEventListener('DOMContentLoaded', () => {
   ];
     
   
+
+  document.getElementById('add').onclick = function() {
+    var appName = prompt("Enter the name of the app:");
+    var appLink = prompt("Enter the link of the app:");
+    var appImgSrc = prompt("Enter the image URL of the app:");
```
[the remark this answers is not in the document]
Comment (rxfe-a) [reply static/assets/scripts/apps.js:296]: it should support all image types. but yeah ill include that0

File: static/assets/scripts/apps.js [hunk 3]
```diff
@@ -289,6 +289,60 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    };
+    var games = JSON.parse(localStorage.getItem("in-custom-a")) || []
+    games.push(app)
+    localStorage.setItem("in-custom-a", JSON.stringify(games));
+    window.location.reload();
+  }
+  
+  document.getElementById('clear').onclick = function() {
+    localStorage.removeItem('in-custom-a');
+  window.location.reload();
```
[the remark this answers is not in the document]
Comment (rxfe-a) [reply static/assets/scripts/apps.js:310]: ill try to think for a better solution

File: static/apps.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (rxfe-a) [reply static/apps.html:30]: like this?
image

[the remark this answers is not in the document]
Comment (rxfe-a) [reply static/apps.html:30]: oh wait nvm i get what your saying

File: static/assets/scripts/apps.js [hunk 4]
```diff
@@ -289,6 +289,60 @@ document.addEventListener('DOMContentLoaded', () => {
...
+  
+   (function() {
+    ga.onclick = function() {
+      images(url);
+    };
+  })();
+  }
+
+  document.body.onload = function() {
+    var appContainer = document.querySelector('.container-apps');
```
Comment (MrBrain295) [inline static/assets/scripts/apps.js:338]:
```suggestion
    const appContainer = document.querySelector('.container-apps');
```

**Comment by rxfe-a** [comment]
ok yeah thanks for the suggestions.
I will work on this more

This will take a while
image
 I will reopen it once I optimize it and make it look better

**Pull request closed without merging**